redis-persistence/src/main/java/com/netflix/conductor/jedis/RedisClusterJedisProvider.java

```diff
@@ -26,10 +34,14 @@ public RedisClusterJedisProvider(HostSupplier hostSupplier){
     @Override
     public JedisCommands get() {
         // FIXME This doesn't seem very safe, but is how it was in the code this was moved from.
-        Host host = new ArrayList<Host>(hostSupplier.getHosts()).get(0);
+        Host host = new ArrayList<>(hostSupplier.getHosts()).get(0);
         GenericObjectPoolConfig poolConfig = new GenericObjectPoolConfig();
         poolConfig.setMinIdle(5);
         poolConfig.setMaxTotal(1000);
-        return new JedisCluster(new HostAndPort(host.getHostName(), host.getPort()), poolConfig);
+
+        JedisPool jedisPool = new JedisPool(poolConfig, host.getHostName(), host.getPort());
+        Jedis jedis = new Jedis();
+        jedis.setDataSource(jedisPool);
+        return jedis;
```

Comment:
doesn't this also need to be "return new JedisCluster(jedisPool);"?

Reply:
ahh good catch, missed this one. Thanks